INSTALL-XAMPP.md

```diff
@@ -0,0 +1,36 @@
+# Install Notes for Windows with XAMPP
+
+It may be handy to install the tools on
+windows for testing and/or developing.
+For such purposes you can install and
+run the tools in cominbation with
+XAMPP.
+
+ 1. Install XAMPP, e.g. in the folder `C:\xampp\`
+ 2. Clone malibu in the `htdocs` subfolder
+ 3. Download jQuery and place it in `.\htdocs\malibu\isbn`
+ 4. Install [`yaz`](http://www.indexdata.com/yaz) for windows.
```

Comment:
workaround may be: copy dll files (not the ic*40.dll files, these are not needed) from \YAZ\bin to .\apache\bin; YAZ may be deinstalled after that
(cf. https://blog.verweisungsform.de/2009-08-17/yaz-php-apache-und-windows/)